sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -624,3 +624,51 @@ export interface RecoverDeletedCertificateOptions extends coreHttp.OperationOpti
  * An interface representing optional parameters for {@link restoreCertificateBackup}.
  */
 export interface RestoreCertificateBackupOptions extends coreHttp.OperationOptions {}
+
+/**
+ * The shape of the contact information for the vault certificates.
+ */
+export interface CertificateContactAll {
+  /**
+   * Email address.
+   */
+  emailAddress: string;
+  /**
+   * Name.
```

Comment:
Is it the user's full name? I'm not a huge fan of these comments since they mostly just repeat the property name ;)